packages/design/sass/_uswds-custom-styles.scss

```diff
@@ -138,3 +138,48 @@ iframe:focus {
   padding-bottom: 62.5%;
   height: 0;
 }
+
+.usa-table {
+  th[scope="row"],
+  th[scope="row"]:nth-child(odd) {
+    background: $color-gray-10;
+  }
+
+  tbody tr:last-child {
+    border-bottom-width: 1px;
+  }
+}
+
+@include at-media("mobile-lg") {
+  .usa-table th[scope="row"],
+  .usa-table th[scope="row"]:nth-child(odd) {
+    background: #fff;
+  }
+
+  .usa-table {
+    thead th {
+      border-bottom-width: 2px;
+
+      &:not(:first-child), &:not(:first-child) {
+        border-left-width: 2px;
+      }
+    }
+
+    tbody {
+      tr:last-child {
+        th, td {
+          border-bottom-width: 1px;
+        }
+      }
+
+      th, td {
+        border-bottom-width: 2px;
+      }
+
+      th:not(:first-child), td:not(:first-child) {
+        border-left-width: 2px;
+      }
+    }
+
+  }
+}
```

Comment:
Is this for parity with the mock-ups? I'd rather we not override the base USWDS classes directly, so we don't leak our styles into the broader app that the library is embedded in.

Reply:
Yes, it's for parity with the mocks. I can add a custom class when I do the full-width UI. Achieving the look that's in the mocks with utility classes isn't possible due to the specificity that's needed, so there was some custom styling applied.

Reply:
Yes, it's for parity with the mocks. Achieving the design with utility classes wasn't possible due to the specificity that's needed. I'll add a custom class when I do the full-width UI.

Reply:
I'm not sure if the differences were intentional or not... unless the differences were added for a specific purpose, my vote would be to stick with the default styling as much as possible.

Reply:
I'll check with the designers, but if it was deliberate, I can build the custom styles into the Astro theme so they aren't distributed with the design package. It might be good to see how someone would work with it from an SDK standpoint.